Tests/APITesters/AllAPITests/SwiftAPITester/OfferingAPI.swift

```diff
@@ -46,19 +46,37 @@ func checkOfferingAPI() {
 }
 
 private func checkCreateOfferingAPI(package: Package) {
+    _ = Offering(
+        identifier: "",
+        serverDescription: "",
+        availablePackages: [package]
```

Comment:
The coloring always confuses me 😅. Very small nitpick but I wonder if we should rename to `rcPackage` to make it less confusing...

Reply:
Updated in c0b38269b10ae605f00bc2aa307fbeaedd677c26